docs/api.rst

```diff
@@ -56,7 +56,7 @@ Models
 
     ShallowFBCSPNet
     Deep4Net
-    EEGInception
+    EEGInceptionERP
```

Comment:
can you add the EEGInceptionMI too here?

Reply:
Done